### PR TITLE
dotCMS/core#22988 Change Traffic Load of experiment

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Adding traffic allocation to experiments. 

https://user-images.githubusercontent.com/31667212/217594416-6b231aa6-7add-4f4e-b216-ca5b5e3aa323.mov

### CODE REVIEW

File: core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_slider.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    border-color: $brand-primary;
+}
+.p-slider.p-slider-animate.p-slider-horizontal .p-slider-handle {
+    transition: background-color 0.2s, color 0.2s, border-color 0.2s, box-shadow 0.2s, left 0.2s;
+}
+.p-slider.p-slider-animate.p-slider-horizontal .p-slider-range {
+    transition: width 0.2s;
+}
+.p-slider.p-slider-animate.p-slider-vertical .p-slider-handle {
+    transition: background-color 0.2s, color 0.2s, border-color 0.2s, box-shadow 0.2s, bottom 0.2s;
```
Comment (alfredo-dotcms) [inline core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_slider.scss:50]: var to DRY?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts [hunk 2]
```diff
@@ -174,6 +174,24 @@ export class DotExperimentsService {
             .pipe(pluck('entity'));
     }
 
+    /**
+     * Set traffic allocation to an experiment
+     * @param {string} experimentId
+     * @param {string} trafficAllocation
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts:180]: fix doc

File: core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_slider.scss (hunk 1 above)
Comment (hmoreras) [reply core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_slider.scss]: done

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts]: done

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        font-weight: $font-weight-semi-bold;
+        font-size: $font-size-large;
+    }
+
+    p {
+        margin-bottom: $spacing-8;
+    }
+}
+
+form input[type="number"] {
```
Comment (oidacra) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.scss:22]: Please add   text-align: center;

**Review by oidacra — CHANGES_REQUESTED** [review]

**Review by oidacra — APPROVED** [review]

**Review by alfredo-dotcms — APPROVED** [review]

**Pull request merged**